debug/legendDataName.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
```

Comment:
I wouldn't add another debug example. It is already horrible to do the smoke tests. This issue is indirectly covered by the guide examples, so I don't think this is necessary

Reply:
Done!